examples/classification/plot_lda_qda.py

```diff
@@ -75,16 +75,15 @@ def plot_data(lda, X, y, y_pred, fig_index):
     alpha = 0.5
 
     # class 0: dots
-    plt.plot(X0_tp[:, 0], X0_tp[:, 1], 'o', alpha=alpha,
-             color='red', markeredgecolor='k')
-    plt.plot(X0_fp[:, 0], X0_fp[:, 1], '*', alpha=alpha,
-             color='#990000', markeredgecolor='k')  # dark red
+    plt.scatter(
```

Comment:
we prefer the visual indentation style that does not break after `(`, as in the former code. Please break before `marker` instead.